iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer_data.hpp

```diff
@@ -69,13 +69,14 @@ class RelativePointerData
     /// @note the maximum offset which can be represented
     static constexpr offset_t MAX_VALID_OFFSET{OFFSET_RANGE - 1U};
     /// @note the maximum allowed size of RelativePointerData
+    // AXIVION Next Line AutosarC++19_03-M11.0.1 : Needs to be verified outside of the class
```

Comment:
I wonder why this rule must not be suppressed in the lines above.

Reply:
Me too. It seems to not be required for the other constexpr

Reply:
Correct. Turns out I was looking at a delta scan only :bulb: